controller/model/posture_check_model_mac.go

```diff
@@ -20,13 +20,19 @@ import (
 	"fmt"
 	"github.com/openziti/edge/controller/persistence"
 	"go.etcd.io/bbolt"
+	"time"
 )
 
 var _ PostureCheckSubType = &PostureCheckMacAddresses{}
 
 type PostureCheckMacAddresses struct {
 	MacAddresses []string
 }
+
+func (p *PostureCheckMacAddresses) LastUpdatedAt(apiSessionId string, pd *PostureData) *time.Time {
+	return nil
```

Comment:
nil on purpose? feels like this needs 'a comment'?

Reply:
It is an interface function, the documentation is on the interface.